chunky/src/res/se/llbit/chunky/ui/Credits.fxml

```diff
@@ -59,11 +59,11 @@
                            <VBox>
                               <children>
                                  <Label text="a_Tom" />
-                                 <Label text="TOGoS" />
                                  <Label text="Redox (alexhliu)" />
+                                 <Label text="TOGoS" />
+                                 <Label text="NotStirred" />
                                  <Label text="jackjt8" />
                                  <Label text="073TheBoss" />
-                                 <Label text="NotStirred" />
                                  <Label text="Ice Selkie (StanleyMines)" />
```

Comment:
Maybe we could generate this automatically from GitHub (at build time) at some point :thinking: